lib/stores/config_store.dart

```diff
@@ -49,6 +49,10 @@ abstract class _ConfigStore with Store {
   @JsonKey(defaultValue: false)
   bool amoledDarkMode = false;
 
+  @observable
+  @JsonKey(defaultValue: true)
+  bool openLinksInExternalApp = true;
```

Comment:
Maybe the default shouldn't be `true`.
The in-app view is already better with my changes. We'll continue to make it not suck.
Let's leave the users to enable this if/when they get frustrated.